client/app/pages/queries/VisualizationEmbed.jsx

```diff
@@ -219,15 +219,15 @@ export default function init(ngModule) {
     return loadSession($route, Auth).then(() => Query.get({ id: $route.current.params.queryId }).$promise);
   }
 
-  ngModule.config($routeProvider => {
-    $routeProvider.when("/embed/query/:queryId/visualization/:visualizationId", {
+  return {
+    "/embed/query/:queryId/visualization/:visualizationId": {
+      template: '<visualization-embed query="$resolve.query"></visualization-embed>',
+      reloadOnSearch: false,
       resolve: {
         query: loadQuery,
       },
-      reloadOnSearch: false,
-      template: '<visualization-embed query="$resolve.query"></visualization-embed>',
-    });
-  });
+    },
+  };
```

Comment:
This syntax enforces Auth (in this case it shouldn't because it's a public route) and is the reason why Visualization embed tests are failing